### ISSUE TITLE
Empty value in end string

### ISSUE THREAD

**Issue by duylong** [logstash-plugins/logstash-filter-kv#91]
Hi,

I can't split this string field : "rev=3&sid="

KV:
```
kv {
        enable_metric => false
        source => [ "[message]" ]
        target => "[test]"
        field_split => "&"
        value_split => "="
        trim_value => '"'
        whitespace => "strict"
}
```
Do you have also this issue?

**Comment by sprytnyserek** [comment]
Yes, I get the same. And from what I can see in kv.rb, it seems intentional, explicit and non-configurable
https://github.com/logstash-plugins/logstash-filter-kv/blob/d010bacd2b8cc585251231ed1acbf2e03c1e8759/lib/logstash/filters/kv.rb#L557
(and the same for "inner kv"). Dunno why... (?) Imo it's not what this plugin is supposed to do.
